.github/workflows/lint.yaml

```
@@ -26,6 +27,8 @@ jobs:
           base: ${{ github.event.pull_request.base.ref || github.event.merge_group.base_ref }}
           ref: ${{ github.event.pull_request.head.ref || github.event.merge_group.head_ref }}
           filters: |
+            has_rfd:
+              - 'rfd/**'
```

```suggestion
              - 'rfd/**.md'
```

This pattern should exclude `cspell.json` and `rfd/assets/**` changes